### DISCUSSION TITLE
filter 기능 관련 질문

### DISCUSSION THREAD

**Discussion by fenoteryn** [bosskmk/pluto_grid#159] [Q&A]
그리드 밖에서 그리드 전체 데이터를 검색하는 기능을 넣고 싶어서 텍스트필드에 값을 입력하면 그리드에 있던 row들이 지워지고 텍스트필드에 입력한 값이 포함된 필드만 새로 추가해 구현하려고 합니다. 텍스트 필드에 값을 입력할 때 그리드에 있던 row들은 
                    stateManager.removeRows(stateManager.rows);로 전부 지워지는데, 이후 
                      stateManager.appendRows(rows);를 사용하여 새로 추가하는 게 안 됩니다. 이유를 알 수 있을까요?

**Comment by bosskmk** [comment]
글쎄요. 말씀하신 내용으로는 문제를 파악하기가 어렵습니다.
문제가 발생 되는 예제 코드가 주시면 확인 해 볼 수 있습니다.
